lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -807,7 +804,7 @@ void FIRRTLModuleLowering::lowerModuleBody(
     // Outputs need a temporary wire so they can be connect'd to, which we
     // then return.
     Value newArg = bodyBuilder.create<WireOp>(
-        port.type, "." + port.getName().str() + ".output");
+        FlipType::get(port.type), "." + port.getName().str() + ".output");
```

Comment:
How does this work?  I thought that wires don't allow an outer flip?

Reply:
This is only looking at output ports at this point. These, by definition, have to have an outer flip. This is the slightly terser alternative to `port.cast<FlipType>().getElementType()`. Perhaps that's more clear? I'm fine to change this post-commit.